includes/class-wc-amazon-payments-advanced-rest-api-controller.php

```diff
@@ -615,8 +609,8 @@ protected function get_missing_authorization_id_request_error( $order_post ) {
 	 * @return null|WP_Error Null if there's no error in the request.
 	 */
 	protected function get_missing_capture_id_request_error( $order_post ) {
-		$ref_id = get_post_meta( $order_post->ID, 'amazon_capture_id', true );
-		if ( ! $ref_id ) {
+		$reference_id = $order_post->get_meta( 'amazon_capture_id', true, 'edit' );
```

Comment:
Similar here: doc-block + naming of `$order_post`.

Reply:
done!